### PR TITLE
[Shell] Remove quoted Identifers

### LINKED ISSUE #29110: Ballerina shell always shows variable names as if they are quoted
**Description:**
Every variable is used in shell as if they are quoted, even if they are not defined as such.
eg:
```
=$ int i = 1

=$ /vars
| ('i) int 'i = 1
```

**Suggested Labels (optional):**
`Area/Shell`

### PR DESCRIPTION
**Pull request by DamithaSenevirathne**
## Purpose

Fixes #29110

Removed unnecessary quotes from the variable declarations as shown in below image.

Screenshot from 2021-09-07 12-30-06

## Check List 
- [x] Read the [Contributing Guide](https://github.com/ballerina-platform/ballerina-lang/blob/master/CONTRIBUTING.md)
- [ ] Updated Change Log
- [ ] Checked Tooling Support (#<Issue Number>)
- [x] Added necessary tests
  - [x] Unit Tests
  - [ ] Spec Conformance Tests
  - [ ] Integration Tests
  - [ ] Ballerina By Example Tests
- [ ] Increased Test Coverage   
- [ ] Added necessary documentation  
  - [ ] API documentation 
  - [ ] Module documentation in Module.md files
  - [ ] Ballerina By Examples

### CODE REVIEW

**Review by mohanvive — APPROVED** [review]
LGTM.

File: ballerina-shell/modules/shell-core/src/test/java/io/ballerina/shell/test/unit/EvaluatorMiscTest.java [hunk 1]
```diff
@@ -68,9 +68,9 @@ public void testEvaluatorImportList() throws BallerinaShellException {
         evaluator.evaluate("import ballerina/lang.'float as prefix2");
         Assert.assertEquals(new HashSet<>(evaluator.availableImports()),
                 Set.of(
-                        "('java) import ballerina/'jballerina.'java as 'java;",
-                        "('prefix) import ballerina/'lang.'int as 'prefix;",
-                        "('prefix2) import ballerina/'lang.'float as 'prefix2;"
+                        "('java) import ballerina/jballerina.'java as 'java;",
```
Comment (pubudu91) [inline ballerina-shell/modules/shell-core/src/test/java/io/ballerina/shell/test/unit/EvaluatorMiscTest.java:71]: Is `java` a keyword? 🤔

Comment (DamithaSenevirathne) [reply ballerina-shell/modules/shell-core/src/test/java/io/ballerina/shell/test/unit/EvaluatorMiscTest.java:71]: No, `java` is not a keyword. In the above case if we use `import ballerina/jballerina.java` output will be `(java) import ballerina/jballerina.java as java;`.

Comment (pubudu91) [reply ballerina-shell/modules/shell-core/src/test/java/io/ballerina/shell/test/unit/EvaluatorMiscTest.java:71]: Ah I see. Is there an explicit quote in the above test case?

Comment (DamithaSenevirathne) [reply ballerina-shell/modules/shell-core/src/test/java/io/ballerina/shell/test/unit/EvaluatorMiscTest.java]: yes, I have updated the above test case without a quote in 214b385683a0cce12446b123aa7f2555f56d3326

Comment (pubudu91) [reply ballerina-shell/modules/shell-core/src/test/java/io/ballerina/shell/test/unit/EvaluatorMiscTest.java]: No, it's fine if that's how it's in the input of the test case.

**Review by pubudu91 — APPROVED** [review]

**Pull request merged**